AK/JsonParser.cpp

```diff
@@ -184,10 +184,15 @@ Optional<JsonValue> JsonParser::parse_number()
             continue;
         }
         if (ch == '-' || (ch >= '0' && ch <= '9')) {
-            if (is_double)
+            if (is_double) {
```

Comment:
As an example, I support your use of curly braces here, since you need them for the `else` anyway and it doesn't add an additional line. In contrast, the curly braces in the inner `if` could and should be removed.